src/bp-activity/bp-activity-notifications.php

```diff
@@ -239,6 +239,16 @@ function bp_activity_at_mention_add_notification( $activity, $subject, $message,
  * @param int                  $commenter_id ID of the user who made the comment.
  */
 function bp_activity_update_reply_add_notification( $activity, $comment_id, $commenter_id ) {
+	// Stop sending notification to user who has muted notifications.
+	if ( $activity->user_id !== $commenter_id ) {
+		$activity_meta = bp_activity_get_meta( $activity->id, 'muted_notification_users' );
```

Comment:
@kishanbuddyboss Create a standalone function for universal use.

Reply:
Fixed